torch/jit/__init__.py

```diff
@@ -283,11 +418,23 @@ def wrapper(func):
         executor_options = {'optimize': True}
         for name in executor_options:
             executor_options[name] = kwargs.pop(name, executor_options[name])
+
+        check_inputs = kwargs.pop('check_inputs', None)
+        disable_checks = kwargs.pop('disable_checks', False)
+        check_tolerance = kwargs.pop('check_tolerance', 1e-7)
```

Comment:
Our tests already have places where we lower the tolerance. I am afraid of false-positives this will introduce (places where the tracer worked before but we report an error now because the tolerance is set too high). Should we just set the tolerance such that all of our examples pass without modification?